internal/pkg/cli/pipeline_deploy.go

```diff
@@ -210,13 +211,19 @@ func (o *deployPipelineOpts) Execute() error {
 	}
 	o.shouldPromptUpdateConnection = bool
 
-	// convert environments to deployment stages
+	// Convert full manifest path to relative path.
+	relPath, err := relPath(o.pipeline.Path)
```

Comment:
do we want the path relative to the current working dir or from the root of the workspace?
I wonder if we should add a `workspace.Rel(o.pipeline.Path)` method that always returns `copilot/path/to/file`

Otherwise, I believe this might give the wrong the wrong path in the cfn template if the user runs `pipeline deploy` from a sub-directory